planning/behavior_path_planner/src/scene_module/pull_over/util.cpp

```diff
@@ -344,6 +68,30 @@ lanelet::ConstLanelets getPullOverLanes(const RouteHandler & route_handler)
   return pull_over_lanes;
 }
 
+bool hasEnoughDistanceToParkingStart(
+  const PathWithLaneId & path, const Pose & current_pose, const Pose & start_pose,
+  const double current_vel, const double maximum_deceleration, const double decide_path_distance)
+{
+  const auto dist_to_start_pose = calcSignedArcLength(
+    path.points, current_pose, start_pose.position, std::numeric_limits<double>::max(), M_PI_2);
+  if (!dist_to_start_pose) return false;
+
+  // const double current_vel = planner_data_->self_odometry->twist.twist.linear.x;
+  const double current_to_stop_distance = std::pow(current_vel, 2) / maximum_deceleration / 2;
+
+  // once stopped, it cannot start again if start_pose is close.
+  // so need enough distance to restart
+  const double eps_vel = 0.01;
```

Comment:
constexpr

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/1911/commits/b47d458c9870cfcb74d01d9866159bd273ee1972